addons/hr_holidays/static/src/js/form/allocation_leave_form.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import viewRegistry from 'web.view_registry';
+
+import FormController from "web.FormController";
+import FormView from "web.FormView";
+
+export const AllocationLeaveFormController = FormController.extend({
+    /**
+     * @override
+     */
+     _applyChanges: async function (dataPointID, changes, event) {
```

Comment:
You can old use the new syntax here :-)

```suggestion
     async _applyChanges(dataPointID, changes, event) {
```

Reply:
Done, thanks :-)